src/Common/src/Interop/Unix/System.Native/Interop.GetPwUid.cs

```diff
@@ -11,6 +11,8 @@ internal static partial class Sys
     {
         internal unsafe struct Passwd
         {
+            internal const int InitialBufferSize = 256;
```

Comment:
I chose to put this const inside of `Passwd` for 2 reasons:

1. It doesn't really belong under `Interop.Sys`, it is more tied to the buffer size for getting Passwd values back.
1. It was a cleaner name in the usage `Interop.Sys.Passwd.InitialBufferSize` vs. `Interop.Sys.InitialPasswdBufferSize`  (debatable)

Let me know what you think